gustaf/utils/arr.py

```diff
@@ -106,10 +106,10 @@ def close_rows(arr, tolerance=None):
     Returns
     --------
     close_stuff: tuple
-      unique_arrays: (n, d) np.ndarray
-      unique_ids: (m) np.ndarray
-      inverse: (n) np.ndarray
-      overlapping: list(list)
+    unique_arrays: (n, d) np.ndarray
+    unique_ids: (m) np.ndarray
+    inverse: (n) np.ndarray
+    overlapping: list(list)
```

Comment:
I meant to state what's in the tuple, but I guess we can either rm `close_stuff` or just state tuple elems as a sentence

Reply:
could we talk about this?